<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Связь с базой данных</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите раздел</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sections"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="choose_term">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите термин</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="terms"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Термин</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="term"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Описание термина</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="term_description"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Раздел термина</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="section_name">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Раздел</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="section"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Описание раздела</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="section_description"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Добавить термин</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="add">
    <property name="title">
     <string>Новый</string>
    </property>
    <addaction name="add_term"/>
    <addaction name="add_section"/>
   </widget>
   <widget class="QMenu" name="change">
    <property name="title">
     <string>Редактировать</string>
    </property>
    <addaction name="change_term"/>
    <addaction name="change_section"/>
   </widget>
   <widget class="QMenu" name="del">
    <property name="title">
     <string>Удалить</string>
    </property>
    <addaction name="del_term"/>
    <addaction name="del_section"/>
   </widget>
   <addaction name="add"/>
   <addaction name="del"/>
   <addaction name="change"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="add_term">
   <property name="text">
    <string>Термин</string>
   </property>
  </action>
  <action name="add_section">
   <property name="text">
    <string>Раздел</string>
   </property>
  </action>
  <action name="change_term">
   <property name="text">
    <string>Термин</string>
   </property>
  </action>
  <action name="change_section">
   <property name="text">
    <string>Раздел</string>
   </property>
  </action>
  <action name="del_term">
   <property name="text">
    <string>Термин</string>
   </property>
  </action>
  <action name="del_section">
   <property name="text">
    <string>Раздел</string>
   </property>
  </action>
  <action name="search_term">
   <property name="text">
    <string>Термина</string>
   </property>
  </action>
  <action name="search_section">
   <property name="text">
    <string>Раздела</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
